libs/core/src/lib/busy-indicator/busy-indicator.component.ts

```diff
@@ -24,6 +24,9 @@ export type BusyIndicatorSize = 's' | 'm' | 'l';
         '[attr.aria-busy]': 'loading',
         '[attr.aria-live]': 'ariaLive',
         '[attr.aria-label]': 'ariaLabel',
+        '[attr.aria-valuemin]': 'ariaValueMin',
+        '[attr.aria-valuemax]': 'ariaValueMax',
+        '[attr.aria-valuenow]': 'ariaValueNow',
```

Comment:
Can you explain me how are these related to the a11y and the component? As per this documentation: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-valuemax_attribute

these values are for a range widget such as a slider, spinbutton or progressbar.

Reply:
@InnaAtanasova per the issue requirements, please see the issue description https://github.com/SAP/fundamental-ngx/issues/6095
![image](https://user-images.githubusercontent.com/3863691/132646006-6d9cc3d4-f149-49c9-b154-dbe913942606.png)

Reply:
This is a busy indicator, not a progress bar.